code/script.js

```diff
@@ -2,334 +2,460 @@
-    img: 'images/josephine.svg',
-    hairColor: 'grey',
-    eyeColor: 'brown',
+    name: 'Lynn',
+    img: 'images2/Lynn.jpg',
+    hatColor: 'dark blue',
+    shirtColor: 'yellow',
     glasses: false,
-    hat: false,
-    smoker: false,
+    moustache: false,
+    scarf: false,
+    unibrow: false
   },
   {
-    name: 'Josh',
-    img: 'images/josh.svg',
-    hairColor: 'yellow',
-    eyeColor: 'green',
+    name: 'Larry',
+    img: 'images2/Larry.jpg',
+    hatColor: 'pink',
+    shirtColor: 'black',
     glasses: false,
-    hat: false,
-    smoker: false,
+    moustache: true,
+    scarf: false,
+    unibrow: false 
   },
-  {
-    name: 'Jude',
-    img: 'images/jude.svg',
-    hairColor: 'black',
-    eyeColor: 'green',
-    glasses: false,
-    hat: false,
-    smoker: false,
+  { 
+    name: 'Lindsey',
+    img: 'images2/Lindsey.jpg',
+    hatColor: 'green',
+    shirtColor: 'white',
+    glasses: true,
+    moustache: false,
+    scarf: false,
+    unibrow: true
   },
   {
-    name: 'Julie',
-    img: 'images/julie.svg',
-    hairColor: 'black',
-    eyeColor: 'brown',
+    name: 'Lenwood',
+    img: 'images2/Lenwood.jpg',
+    hatColor: 'black',
+    shirtColor: 'yellow',
     glasses: true,
-    hat: true,
-    smoker: false,
+    moustache: true,
+    scarf: false,
+    unibrow: false
   },
 ]
 
 // Global variables
-let secret, currentQuestion, charactersInPlay
+
+let currentQuestion 
+
+let secret  
 
 // Draw the game board
-const generateBoard = () => {
+function generateBoard() {
   board.innerHTML = ''
   charactersInPlay.forEach((person) => {
     board.innerHTML += `
-      <div class="card">
-        <p>${person.name}</p>
-        <img src=${person.img} alt=${person.name}>
-        <div class="guess">
+      <div class='card'>
+        <img class='person-image' src=${person.img} alt=${person.name}>
+        <div class='guess'>
           <span>Guess on ${person.name}?</span>
-          <button class="filled-button small" onclick="guess('${person.name}')">Guess</button>
+          <button class='small' onclick="guess('${person.name}')">Guess</button>
         </div>
+        <p>${person.name}</p>
+        <button class='small-mobile' onclick="guess('${person.name}')">Guess</button>
       </div>
     `
   })
 }
 
+function playAgain() {
+  winOrLoseSection.style.display = 'none';
+  winOrLoseText.innerHTML = '';
+  board.style.display = 'flex';
+  start();
+
+}
+
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
 }
 
 // This function to start (and restart) the game
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
-  charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  charactersInPlay = CHARACTERS;
+  generateBoard();
+  setSecret();
+  console.log(secret);
```

Comment:
We dont have any needs for the console.log after the game is finnished. Because it looks like your **secret** work.

Reply:
Fixed it!

Reply:
Yes, remember to always clean your code from all console.log()s and commented out code blocks and tips left for you by teachers 🗡️ ✂️